portfolio/src/main/webapp/index.html

```diff
@@ -71,10 +71,11 @@
             <input type="text" name="email" placeholder="Enter Your Email">
 
             <label>Message: </label>
```

Comment:
Are the Name, Email, and Message still going to be used? As it is it will be confusing for site visitors when they input this information and the Submit button doesn't do anything with them. Should they be in the form with the "title"? Are you planning to store the information anywhere?